packages/services/src/nbconvert/index.ts

```diff
@@ -76,6 +81,32 @@ export class NbConvertManager {
     return this._exportFormats;
   }
 
+  /**
+   * Export a notebook to a given format.
+   *
+   * @param options - The export options.
+   * @param options.format - The export format (e.g., 'html', 'pdf').
+   * @param options.path - The path to the notebook to export.
+   * @param options.download - Whether to download the file or open it in a new tab. Defaults to false.
+   */
+  async export(options: {
```

Comment:
given `export` has some overloaded meaning in ESM/TS, and this is intended to be overloaded, maybe it's worth some more tweaks:

- consider a slightly more distinguishing name, e.g. `exportAs` 
- make the options an `IExportOptions` so that overloads have something concrete to point at instead of a list of positional args
- potentially allow per-`options.exporterOptions` (`download?` might just be the first example in the simple case)

Reply:
Thanks for the review 👍 

I pushed a couple of changes to implement these ideas.